packages/cli-test/src/cli/commands/datastore.ts

```diff
@@ -14,6 +14,10 @@ export interface DatastoreCommandArguments {
   queryExpressionValues: object;
 }
 
+function escapeJSON(obj: Record<string, unknown>): string {
+  return `"${JSON.stringify(obj).replace(/"/g, '\\"')}"`;
```

Comment:
This in combination with using using `child_process` `spawn` et al's `args` array of arguments helped with dealing with JSON encoding of command line arguments in a cross-platform way.

Reply:
It may be worthwhile leaving a code comment to explain that this is to help support cross-platform JSON encoding